build.xml

```diff
@@ -30,6 +30,7 @@
     <property environment="env"/>
     <property file="build.properties" />
     <property file="build.properties.default" />
+    <property file="${user.home}/.ant/build.properties"/>
```

Comment:
do we need this ?

Reply:
yes

Reply:
longer answer - yes, it is ported from newer branches (5.0) and it lets the user to define global properties, like tmp / cache directories or nvd api key

Reply:
Jacek has worktrees per branch, I have a worktree per logical branch (I have cassandra-{3.0,3.11,4.0,4.1,5.0,trunk} directories and then I just switch branches for that particular branch inside). `build.properties` are `.gitignor`-ed so that means that these properties would be "lost" in Jacek's case, but not mine. We also do not want to hardcode paths outside of repo except of some dirs in `tmp.dir` nor anywhere in user's home. Somebody is OK with this, somebody is not ... to each their own. This one solves it globally for everybody how they please.